tests/formats/test_bmsmd.py

```diff
@@ -0,0 +1,11 @@
+from tests.test_lib import parse_and_build_compare
+
+from mercury_engine_data_structures.formats.bmsmd import Bmsmd
+from mercury_engine_data_structures.game_check import Game
+
+
+def test_bmsmd(samus_returns_path):
+    file_path = samus_returns_path.joinpath(r"gui\minimaps\c10_samus.bmsmd")
+    parse_and_build_compare(
```

Comment:
Can you use the `parse_build_compare_editor` here instead? Then the uncovered `construct_class` method would be in the tests, too.